client/src/components/core/ms-types.ts

```diff
@@ -12,3 +12,35 @@ export enum MsModalResult {
   Cancel = 'cancel',
   Confirm = 'confirm',
 }
+
+export interface MsOption {
+  label: string,
+  key: any,
+  disabled?: boolean,
+}
+
+export function getMsOptionByKey(options: MsOption[], key: any): MsOption | undefined {
+  return options.find((option) => {
+    return option.key === key;
+  });
+}
+
+export type MsDropdownOption = MsOption;
+export type MsSorterOption = MsOption;
+
+export interface MsChangeEvent {
```

Comment:
Could remove the export here, noone is supposed to use `MsChangeEvent` directly.